files_to_prompt/cli.py

```diff
@@ -271,15 +283,23 @@ def cli(
         ---
         ...
 
+    Use `--no-filename` to hide the filenames in the output.
+
     If the `--cxml` flag is provided, the output will be structured as follows:
 
     \b
         <documents>
-        <document path="path/to/file1.txt">
+        <document index="1">
```

Comment:
💬 This was updated because the documentation didn't follow what the implementation currently does.